src/jdk.jdeps/share/classes/com/sun/tools/javap/JavapTask.java

```diff
@@ -860,6 +861,17 @@ private JavaFileObject getClassFileObject(String className) throws IOException {
             if (moduleLocation != null) {
                 fo = fileManager.getJavaFileForInput(moduleLocation, className, JavaFileObject.Kind.CLASS);
             } else {
+                if (className.indexOf('.') > 0) try {
```

Comment:
I would put the try into a block - it is slightly longer, but much clear, I think.

Reply:
Fixed, thanks.